common/lfdemod.c

```diff
@@ -1067,6 +1089,23 @@ void psk1TOpsk2(uint8_t *BitStream, size_t size)
 	return;
 }
 
+// by marshmellow
+// convert psk2 demod to psk1 demod
+// from only transition waves are 1s to phase shifts change bit
+void psk2TOpsk1(uint8_t *BitStream, size_t size)
```

Comment:
I'll have to look closer, might be a bug, I might not have tested the psk2 Sim (which is the only reason for this)